src/main/resources/docaccordion.css

```diff
@@ -3,4 +3,8 @@
   display: none;
   font-style: italic;   
   padding-top: 5px;
-}
\ No newline at end of file
+}
+.xwiki-accordion-content ul {
+  list-style-type: disc;
+  padding-left: 2.5em;
+}
```

Comment:
The content, that could represent the content of any page, could contain also other `list-style-type`s and there are a lot. Applying this rule means that any user list created in a page content will be displayed in the accordion as a disc list style type which is not right.